charts/datadog/README.md

```diff
@@ -693,9 +693,9 @@ helm install <RELEASE_NAME> \
 | datadog.systemProbe.enableConntrack | bool | `true` | Enable the system-probe agent to connect to the netlink/conntrack subsystem to add NAT information to connection data |
 | datadog.systemProbe.enableDefaultKernelHeadersPaths | bool | `true` | Enable mount of default paths where kernel headers are stored |
 | datadog.systemProbe.enableDefaultOsReleasePaths | bool | `true` | enable default os-release files mount |
-| datadog.systemProbe.enableKernelHeaderDownload | bool | `true` | Enable the downloading of kernel headers for runtime compilation of eBPF probes |
+| datadog.systemProbe.enableKernelHeaderDownload | string | `nil` | Enable the downloading of kernel headers for runtime compilation of eBPF probes |
```

Comment:
Not ideal that this lists the type as `string`, but not sure what we can do about it.

Reply:
@clamoriniere any advice on how to approach this?